### ISSUE TITLE
bibo:Image both used in "medium" and "type"

### ISSUE THREAD

**Issue by acka47** [lobid/lodmill#785]
See [morph, lines 233-238](https://github.com/lobid/lodmill/blob/9710b449d88e981dc8080e2a9327a1c4dc5f1966/lodmill-rd/src/main/resources/morph-hbz01-to-lobid.xml#L233-L238). As said in https://github.com/lobid/lodmill/issues/512#issuecomment-54944812, `bibo:Image` should only be used with dct:medium.

**Comment by ChristophEwertowski** [comment]
Not shown in frontend but in data ([query](http://lobid.org/resources/search?t=image))(12260 hits).

**Comment by ChristophEwertowski** [comment]
Did the commit already (see https://github.com/hbz/lobid-resources/pull/547) but: What publication type the maps are going to be?

**Comment by acka47** [comment]
> What publication type the maps are going to be?

I don't see what this question ha to do with the issue. Currently, only 15 of the ~13k resources typed with `Image` actually have `Map` as medium, see http://lobid.org/resources/search?q=type%3AImage&medium=http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2FMap. Or did you mean "What publication type the _images_ are going to be?"

**Comment by ChristophEwertowski** [comment]
Sorry, I got mixed up because the same question concerns both images and maps. At the moment we have many images which are typed as books although they are "Posters". But that are questions for a different ticket.
Added HT017066705 as a test resource. I will assign this ticket to you when @dr0i and I have found out why the smalltest index doesn't include new resources ([HT017066705 (production)](http://lobid.org/resources/HT017066705.json) / [HT017066705 (test)](http://test.lobid.org/resources/HT017066705.json)

Update: They were included but ".json"-links didn't work because of an old play version.
